client.go

```diff
@@ -1030,24 +1030,25 @@ func (c *Client) ReadWithContext(ctx context.Context, req *ua.ReadRequest) (*ua.
 	err := c.SendWithContext(ctx, req, func(v interface{}) error {
 		err := safeAssign(v, &res)
 
+		if err != nil {
+			return err
+		}
```

Comment:
```suggestion
		err := safeAssign(v, &res)
		if err != nil {
			return err
		}
```

I don't think Go devs not usually put a blank line here (or even do like `if err := safeAssign(v, &res); err != nil { ... }`).